src/main/java/io/github/pylonmc/pylon/base/content/tools/LumberAxe.java

```diff
@@ -28,7 +28,7 @@ public LumberAxe( @NotNull ItemStack stack) {
         super(stack);
     }
 
-    private final Set<Event> eventsToIgnore = HashSet.newHashSet(0);
+    private static final Set<Event> eventsToIgnore = HashSet.newHashSet(0);
```

Comment:
This may leak memory if the events are never removed. Also why not `new HashSet><>()`?